presto-cassandra/pom.xml

```diff
@@ -14,6 +14,16 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
+    
+    <dependencyManagement>
```

Comment:
Formatting looks off--looks like indented with tabs instead of spaces?

Reply:
should be fixed now